PhysicsTools/SelectorUtils/interface/VersionedIdProducer.h

```diff
@@ -86,7 +86,7 @@ VersionedIdProducer(const edm::ParameterSet& iConfig) {
     }
 
     if( idMD5 != calculated_md5 ) {
-      edm::LogError("IdConfigurationNotValidated")
+      edm::LogInfo("IdConfigurationNotValidated")
```

Comment:
@lgray 
please note

If I understand correctly, with md5 check machinery not working, a noticeable feature of VIDs to be able to validate the configuration is dead.